src/client/src/test/JobPosting.test.js

```diff
@@ -0,0 +1,18 @@
+import JobPosting from "../components/JobListing";
+import '@testing-library/jest-dom';
+import React from "react";
+import { render, screen, fireEvent, cleanup } from "@testing-library/react";
+
+
+test("jobListing page renders properly", ()=>{
+    render(<JobPosting/>);
+
+    const fnameTextboxElement = screen.getByRole('textbok', {name:'fname'});
```

Comment:
This test failed on my system for this reason:
`TestingLibraryElementError: Unable to find an accessible element with the role "textbok" and name "fname"`